tee-worker/ts-tests/package.json

```diff
@@ -5,9 +5,12 @@
 		"gen-meta": "yarn gen-meta:sidechain && yarn gen-meta:parachain",
 		"gen-meta:sidechain": "../bin/integritee-cli print-sgx-metadata-raw > litentry-sidechain-metadata.json",
 		"gen-meta:parachain": "curl -s -H \"Content-Type: application/json\" -d '{\"id\":\"1\", \"jsonrpc\":\"2.0\", \"method\": \"state_getMetadata\", \"params\":[]}' http://localhost:9933 > litentry-parachain-metadata.json",
-		"gen-type": "yarn gen-type:defs && yarn gen-type:meta && yarn format",
-		"gen-type:defs": "ts-node --skip-project node_modules/.bin/polkadot-types-from-defs  --package . --input ./interfaces --endpoint ./litentry-sidechain-metadata.json",
-		"gen-type:meta": "ts-node --skip-project node_modules/.bin/polkadot-types-from-chain --package . --output ./interfaces --endpoint ./litentry-sidechain-metadata.json",
+		"gen-type:parachain": "yarn gen-type-parachain:defs && yarn gen-type-parachain:meta && yarn format",
+		"gen-type-parachain:defs": "ts-node --skip-project node_modules/.bin/polkadot-types-from-defs  --package . --input ./parachain-interfaces --endpoint ./litentry-parachain-metadata.json",
+		"gen-type-parachain:meta": "ts-node --skip-project node_modules/.bin/polkadot-types-from-chain --package . --output ./parachain-interfaces --endpoint ./litentry-parachain-metadata.json",
+		"gen-type:sidechain": "yarn gen-type-sidechain:defs && yarn gen-type-sidechain:meta && yarn format",
+		"gen-type-sidechain:defs": "ts-node --skip-project node_modules/.bin/polkadot-types-from-defs  --package . --input ./sidechain-interfaces --endpoint ./litentry-sidechain-metadata.json",
+		"gen-type-sidechain:meta": "ts-node --skip-project node_modules/.bin/polkadot-types-from-chain --package . --output ./sidechain-interfaces --endpoint ./litentry-sidechain-metadata.json",
```

Comment:
Do we have these documented somewhere? If so then that probably needs updated as well :thinking: